src/main/assembly/assembly-offline.xml

```diff
@@ -51,6 +51,16 @@
             <includes>
                 <include>**</include>
             </includes>
+            <excludes>
```

Comment:
The same changes must be applied also to [assembly-no-index.xml](https://github.com/windup/windup-distribution/blob/master/src/main/assembly/assembly-no-index.xml) file

Reply:
That's been done now